linchpin/ansible.cfg

```diff
@@ -0,0 +1,4 @@
+[defaults]
```

Comment:
Do we want these too?
inventory = /ansible/inventory
ansible_force_color = true

Reply:
Will the inventory property change any functionality? I just copied the config from the sample project.